ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/product/v1/product/product.html

```diff
@@ -26,8 +27,10 @@
           data-locale="${page.language}"
           data-load-client-price="${product.loadClientPrice}">
         <sly data-sly-test.found="${product.found}">
+            <sly data-sly-test.isGroupedProduct="${product.isGroupedProduct}" />
```

Comment:
`data-sly-test.isGroupedProduct="${product.isGroupedProduct}"` can be added to the previous `<sly>` tag.

Reply:
No that doesn't work, it would otherwise enable/disable the display of that entire block based on grouped products, and that's not the intention here. I added a separate `sly` tag here because I basically want to set the `isGroupedVariable` but I cannot do that at line 33 because it there "negates" the test with `!`. Which I'd have to set a variable `isNotGroupedProduct`, and that would make all further tests more complicated like "double not" with `${!isNotGroupedProduct}`.

Reply:
Ok got it